iceoryx_utils/test/moduletests/test_relative_pointer.cpp

```diff
@@ -501,13 +484,12 @@ TEST_F(RelativePointer_test, memoryRemapping)
 
 TEST_F(RelativePointer_test, MemoryReMapping_SharedMemory)
 {
-    MemMap memMapWriter(reinterpret_cast<void*>(BaseAddress1), this->m_fileDescriptor);
-    MemMap memMapReader(reinterpret_cast<void*>(BaseAddress2), this->m_fileDescriptor);
+    MemMap memMapWriter(this->m_fileDescriptor);
+    MemMap memMapReader(this->m_fileDescriptor);
 
-    Data* dataPointer1 = new (reinterpret_cast<void*>(BaseAddress1)) Data(12, 21);
+    Data* dataPointer1 = new (memMapWriter.getMappedAddress()) Data(12, 21);
 
-    EXPECT_EQ(reinterpret_cast<uintptr_t>(dataPointer1), BaseAddress1);
-    EXPECT_EQ(dataPointer1->Data1, reinterpret_cast<Data*>(BaseAddress2)->Data1);
+    EXPECT_EQ(dataPointer1->Data1, reinterpret_cast<Data*>(memMapReader.getMappedAddress())->Data1);
 
     int offset = ShmSize / 2;
     auto offsetAddr1 = reinterpret_cast<int*>(static_cast<uint8_t*>(memMapWriter.getMappedAddress()) + offset);
```

Comment:
Looks ok to me (the main change is that the tests are now independent of actual addresses of the mapped memory). However, I plan to rework the implementation of the relative pointer as well as the tests (the core functionality will not change)